apps/pom.xml

```diff
@@ -933,7 +933,7 @@
             <plugin>
                 <groupId>org.codehaus.mojo</groupId>
                 <artifactId>versions-maven-plugin</artifactId>
-                <version>2.11.0</version>
+                <version>2.16.1</version>
```

Comment:
Does this plugin need to be updated for this to work?

Reply:
This upgrade removed the validation warning for this particular. The upgrade was done to remove these warnings as these versions might not be compatible in the future with Maven version 4.0+. I reverted the previous comment as that did not remove the validation warnings.

```
[WARNING] 
[WARNING] Plugin validation issues were detected in 3 plugin(s)
[WARNING] 
[WARNING]  * org.apache.maven.plugins:maven-site-plugin:3.4
[WARNING]  * org.codehaus.mojo:versions-maven-plugin:2.11.0
[WARNING]  * org.xolstice.maven.plugins:protobuf-maven-plugin:0.6.1
[WARNING] 
```

Reply:
I think this is a really good intention, but as discussed, let's keep this PR focused on the changes that help us realize code coverage report generation/upload. The various upgrades can be handled in [BFD-2770](https://jira.cms.gov/browse/BFD-2770). Might be worthwhile to comment on that story to call this specific upgrade out 👍